### ISSUE TITLE
Liquid syntax error when accessing data objects where ID starts with a number

### ISSUE THREAD

**Issue by jaybrueder** [contentful/jekyll-contentful-data-import#57]
Hi,

I am getting warning from Jekyll when accessing data objects where the ID begins with a number.

I am using them like this:

```
{% assign contentful_data = site.data.contentful.spaces.company.course.6MAMEasi0EiqKWSKeSaUIs %}
```

And I see this warning in the CLI output when running `jekyll serve`:

```
Liquid Warning: Liquid syntax error (line 1): Expected id but found number in "{{site.data.contentful.spaces.company.course.6MAMEasi0EiqKWSKeSaUIs }}" in index.html
```

No problem when the ID starts with a letter.

Maybe we could define a specific content model field as the ID?

**Comment by dlitvakb** [comment]
Answered on #56.

Cheers

**Comment by jaybrueder** [comment]
Solved this with the help in #56 

I have organisied my data into one YAML file instead of multiples and iterate over them, selecting the fitting dataset like so:

```
---
layout: default
title: CourseName
---

{% for item in site.data.contentful.spaces.company.course %}
    {% if item.title == page.title %}
        {% assign contentful_data = item %};
        {% break %}
    {% endif %}
{% endfor %}

<h1>{{ contentful_data.title }}</h1>
<p>{{ contentful_data.description }}</p>
```

I am not accessing the object directly through the ID as before, but rather find the correct dataset by checking the title attribute.
